### PR TITLE
[ADD] sale_order_report_with_note (migrated sale_order_webkit to v9 in qweb and renamed the module)

### PR DESCRIPTION
**Pull request by serpentcs-dev1**

### CODE REVIEW

File: sale_order_report_with_note/README.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+=============================
+
+* This module replaces the webkit Quotation / Sale Order report by
+  a qweb report and adds header and footer notes on the sale order 
+  and quotation
+
+
+Installation
+============
+
```
Comment (AaronHForgeFlow) [inline sale_order_report_with_note/README.rst]: Please remove empty sections

File: sale_order_report_with_note/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=============================
+Sale Order Report With Note
+=============================
+
+* This module replaces the webkit Quotation / Sale Order report by
```
Comment (AaronHForgeFlow) [inline sale_order_report_with_note/README.rst]: Not sure if this description is correct as the report already exists in qweb. Actually it adds the notes to the existing report

**Comment by rousseldenis** [comment]
@serpentcs-dev1 Could we consider this too old and close ?

**Comment by rousseldenis** [comment]
@JayVora-SerpentCS

**Comment by JayVora-SerpentCS** [comment]
Close plz

**Pull request closed without merging**